### ISSUE TITLE
Schema View throwing errors

### ISSUE THREAD

**Issue by antares1470** [constellation-app/constellation#2054]
### Prerequisites

- [x] Put an X between the brackets on this line if you have done all of
    the following:

    -   Running the latest version of Constellation

    -   Attached the ***Support Package*** via `Help` &gt;
        `Support Package`

    -   Checked the FAQs:
        https://github.com/constellation-app/constellation/wiki/FAQ

    -   Checked that your issue isn’t already filed:
        https://github.com/constellation-app/constellation/issues

    -   Checked that there is not already a module that provides the
        described functionality:
        https://github.com/constellation-app/constellation/wiki/Catalogue-of-Repositories

### Description

When trying to open Schema View, it freezes for a moment and then periodically throws a bunch of exceptions (see comments for some of them).

### Steps to Reproduce

1. Open new graph
2. Open Schema View
3. Wait

**Expected behaviour:** Schema View loads and runs without issues

**Actual behaviour:** Schema View doesn't load properly and throws a bunch of errors

**Reproduces how often:** \[What percentage of the time does it reproduce?\]

### Additional Information

See comments for sample of exceptions thrown. It seems they are all related but possibly come from the use of the use of the indriya library

**Comment by antares1470** [comment]
```
java.lang.ExceptionInInitializerError: Exception java.lang.IncompatibleClassChangeError: Method 'tech.units.indriya.function.MultiplyConverter tech.units.indriya.function.MultiplyConverter.ofPiExponent(int)' must be Methodref constant [in thread "JavaFX Application Thread"]
	at si.uom.NonSI.<clinit>(NonSI.java:129)
	at org.geotools.referencing.cs.DefaultCoordinateSystemAxis.<clinit>(DefaultCoordinateSystemAxis.java:107)
	at org.geotools.referencing.cs.AbstractCS.<clinit>(AbstractCS.java:79)
	at org.geotools.referencing.crs.DefaultEngineeringCRS.<clinit>(DefaultEngineeringCRS.java:78)
	at org.geotools.referencing.factory.epsg.CartesianAuthorityFactory.<clinit>(CartesianAuthorityFactory.java:54)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(MethodHandleAccessorFactory.java:300)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(MethodHandleAccessorFactory.java:103)
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(ReflectionFactory.java:200)
	at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Constructor.java:549)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at java.base/java.util.ServiceLoader$ProviderImpl$2.run(ServiceLoader.java:797)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:802)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:729)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1403)
	at org.geotools.util.factory.FactoryRegistry.register(FactoryRegistry.java:967)
	at org.geotools.util.factory.FactoryRegistry.scanForPlugins(FactoryRegistry.java:873)
	at org.geotools.util.factory.FactoryRegistry.scanForPluginsIfNeeded(FactoryRegistry.java:903)
	at org.geotools.util.factory.FactoryRegistry.getFactories(FactoryRegistry.java:265)
	at org.geotools.referencing.ReferencingFactoryFinder.getFactories(ReferencingFactoryFinder.java:183)
	at org.geotools.referencing.ReferencingFactoryFinder.getCRSAuthorityFactories(ReferencingFactoryFinder.java:453)
	at org.geotools.referencing.DefaultAuthorityFactory.getBackingFactory(DefaultAuthorityFactory.java:83)
Caused: java.lang.NoClassDefFoundError: Could not initialize class org.geotools.referencing.cs.DefaultCoordinateSystemAxis
	at org.geotools.referencing.factory.TransformedAuthorityFactory.<clinit>(TransformedAuthorityFactory.java:105)
	at org.geotools.referencing.factory.epsg.DirectEpsgFactory.<init>(DirectEpsgFactory.java:525)
	at org.geotools.referencing.factory.epsg.FactoryUsingSQL.<init>(FactoryUsingSQL.java:51)
	at org.geotools.referencing.factory.epsg.FactoryUsingAnsiSQL.<init>(FactoryUsingAnsiSQL.java:119)
	at org.geotools.referencing.factory.epsg.hsql.FactoryUsingHSQL.<init>(FactoryUsingHSQL.java:51)
	at org.geotools.referencing.factory.epsg.hsql.ThreadedHsqlEpsgFactory$1.<init>(ThreadedHsqlEpsgFactory.java:322)
	at org.geotools.referencing.factory.epsg.hsql.ThreadedHsqlEpsgFactory.createBackingStore(ThreadedHsqlEpsgFactory.java:322)
	at org.geotools.referencing.factory.epsg.ThreadedEpsgFactory.createBackingStore0(ThreadedEpsgFactory.java:367)
	at org.geotools.referencing.factory.epsg.ThreadedEpsgFactory.createBackingStore(ThreadedEpsgFactory.java:388)
	at org.geotools.referencing.factory.DeferredAuthorityFactory.getBackingStore(DeferredAuthorityFactory.java:120)
	at org.geotools.referencing.factory.BufferedAuthorityFactory.isAvailable(BufferedAuthorityFactory.java:245)
	at org.geotools.referencing.factory.DeferredAuthorityFactory.isAvailable(DeferredAuthorityFactory.java:101)
	at org.geotools.util.factory.FactoryRegistry.isAvailable(FactoryRegistry.java:761)
	at org.geotools.util.factory.FactoryRegistry.isAcceptable(FactoryRegistry.java:584)
	at org.geotools.util.factory.FactoryRegistry.lambda$getFactories$0(FactoryRegistry.java:267)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
	at java.base/java.util.Spliterators$IteratorSpliterator.tryAdvance(Spliterators.java:1950)
	at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.lambda$initPartialTraversalState$0(StreamSpliterators.java:292)
	at java.base/java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.fillBuffer(StreamSpliterators.java:206)
	at java.base/java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.doAdvance(StreamSpliterators.java:161)
	at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.tryAdvance(StreamSpliterators.java:298)
	at java.base/java.util.Spliterators$1Adapter.hasNext(Spliterators.java:681)
	at org.geotools.util.LazySet.isEmpty(LazySet.java:100)
	at org.geotools.referencing.factory.ManyAuthoritiesFactory.<init>(ManyAuthoritiesFactory.java:131)
	at org.geotools.referencing.DefaultAuthorityFactory.getBackingFactory(DefaultAuthorityFactory.java:110)
	at org.geotools.referencing.DefaultAuthorityFactory.<init>(DefaultAuthorityFactory.java:63)
	at org.geotools.referencing.CRS.getAuthorityFactory(CRS.java:235)
	at org.geotools.referencing.CRS.decode(CRS.java:507)
	at org.geotools.referencing.CRS.decode(CRS.java:428)
	at au.gov.asd.tac.constellation.utilities.geospatial.Shape$SpatialReference.getSrs(Shape.java:177)
	at au.gov.asd.tac.constellation.plugins.importexport.geospatial.SpatialReferenceParameterValue.toString(SpatialReferenceParameterValue.java:97)
	at au.gov.asd.tac.constellation.plugins.parameters.types.SingleChoiceParameterType$SingleChoiceParameterValue.toString(SingleChoiceParameterType.java:534)
	at au.gov.asd.tac.constellation.plugins.parameters.PluginParameter.getStringValue(PluginParameter.java:526)
	at au.gov.asd.tac.constellation.views.schemaview.providers.PluginsNodeProvider.lambda$setContent$7(PluginsNodeProvider.java:173)
```

**Comment by capricornunicorn123** [comment]
I've done some work on this issue to explore some possible solutions. Here is what I've found:

- The org="org.languagetool" name="language-en" rev="6.0" dependency pulls in a number of jar files that are superseded versions of existing jar files in Core Dependencies. 
- Some packages that use these Jar files struggle to reference the correct version. Currently this is just spell check and the schema view but I suspect we would be vulnerable to more creeping in as time progresses and packages update. (I don't know how maven resolves dependency versions as I figure multi version packages would be quite common in large projects)
- In the case of Indriya, the schema view wants 2.0.2 and Spell check wants 1.3. Neither of these constellation components can use the others version.
- Spell check is happy to work if both Jar files are included in the dependencies, the Schema view is not (so they must both be using 1.3). 
- Excluding the Indriya 1.3 jar from the core dependency update and including a static copy in CoreUtilities does not work as the static Indriya 1.3 Jar in Core Utilities is unable to arbitrarily reference other Jar files in Core Dependencies.

The solution in #2069 includes all of the language tools dependencies statically in a separate module that is independent from Core Dependencies, all of the spell check functionality is moved to that module. It's not a great solution and an improvement would be to make these dependencies updatable in the build scripts as is the case with Core Dependencies. I don't know how to do this...

**Comment by antares1470** [comment]
@capricornunicorn123 I mentioned in a comment on your pull request but I'll mention it here as well: 
I had done some testing locally with removing conf from the language-en dependency entry in ivy.xml which meant for that library it only downloaded jars that we didn't already have (indriya still had conf). The result from testing from what I could see was no difference (i.e. spellcheck worked, schema view didn't); suggesting that the only library really requiring more than one instance is indriya and every other library can work with newer versions.

**Comment by capricornunicorn123** [comment]
@antares1470 I'm bumping around in the dark a bit on this ticket as I'm not familiar with this level of dependency management. It's good to know that we can eliminate those other superseded dependencies. I'm not sure how we get around the problem of the arbitrary referencing issues (it may be me incorrectly moving JAR's around). The Indriya 1.3 library is referenced from the language-core Jar and not from Constellation's spell-checking utilities directly. So I'm assuming we have to consider the language tool JARs if/ when we're moving the Indriya 1.3 JAR. i.e we can't move Indriya 1.3 to a module that is dependent on Core Dependencies as the languagetool JARs in Core Dependencies won't be able to reference Indriya 1.3. I've tried moving the languagetool JARs and the Indriya 1.3 JAR to Core Utilities however I either get issues where the JLanguageTool doesn't build correctly, or Language Tool has issues "arbitrarily referencing" another JAR in Core Dependencies from Core Utilities.

**Comment by antares1470** [comment]
Going to close this ticket since this issue is now out of master but noting that the issue will need to be resolved for #2088 in order for that PR to be merged
